### DISCUSSION TITLE
Can't get Incomplete task notifier to work

### DISCUSSION THREAD

**Discussion by chhayse7en** [anasty17/mirror-leech-telegram-bot#1168] [Q&A] [answered]
I have enabled INCOMPLETE_TASK_NOTIFIER = "true", and filled in the generated MongoDB correctly by following the instruction in README.md. The log show "bot - INFO - Users data has been imported from Database" when restart the bot but I don't receive any notifications about the incomplete tasks from previous session. I tested it by restarting the bot during downloading process.

**Comment by anasty17** [comment]
U should edit it from bot settings cmd if u added databases url. View the settings if its false or true

**Reply by chhayse7en** [reply]
It's true in botsetting. Database url filled correctly as the log show "bot - INFO - Users data has been imported from Database"  but still not give any notifications after restart the bot during downloading process. By the way, I use the bot directly without adding it to supergroup.

**Reply by anasty17** [reply]
First user data not related to bot settings, it's only tob sers settings.
2nd I wrote in readme incomplete task notifier works only in supergroup. Any message u need to get link for it then use supergroup

**Comment by anasty17** [comment, marked as answer]
![20221026_195109](https://user-images.githubusercontent.com/77075674/198087233-d295ce27-2c19-45a9-8502-0211437c2eea.jpg)
Read and don't ask again before reading. And the confusing thing that u already said it (not in supergroup) but idk why u asking me! Only confusing me or what!

**Reply by chhayse7en** [reply]
Sorry, I'm just frustrated that I can't get it to work. Even when adding it to the group, filled in MongoDB, and set INCOMPLETE_TASK_NOTIFIER = "true". But still I can't just get it to work. One more thing, I've never asked before reading. I've read enough times and follow every step carefully. It's just I'm lost now.

**Reply by anasty17** [reply]
Even in supergroup not working ? U enabled it before adding download ?

**Reply by chhayse7en** [reply]
I can get it to work now by changing the group from private to public and then change it back to private again. Thank you for taking your time to go through this problem with me.

**Reply by anasty17** [reply]
Bruh! Supergroup not like group!
